src/components/Auth/SignUp.jsx

```diff
@@ -8,149 +8,153 @@ import { useTranslation } from 'react-i18next';
 import { authApi } from '../../api/authApi';
 
 const SignUp = () => {
-	const { t } = useTranslation()
-	const { auth } = useAuth()
-	const navigate = useNavigate();
-	const [errorMessage, setErrorMessage] = useState(null);
-	const [visibilityToggle, setvisibilityToggle] = useState(false);
-	document.title = t('signup') + " | " + process.env.REACT_APP_APP_NAME
-	
-	const onSubmitHandler = async (loginData) => {
-		authApi.signUp(loginData)
-			.then(res => {
-				if (res.data.status === "success") {
-					navigate('/signin')
-				} else if (res.data.status === 'error') {
-					setErrorMessage(res.data.message.email)
-				}
-			})
-	}
+    const { t } = useTranslation()
+    const { auth } = useAuth()
+    const [successMessage, setSuccessMessage] = useState(null);
+    const [errorMessage, setErrorMessage] = useState(null);
+    const [visibilityToggle, setvisibilityToggle] = useState(false);

```

Comment:
change name of function ''setvisibilityToggle'' to "setVisibilityToggle" it should be camel case

Reply:
@hatik92 had you do anything about this?